src/AccessibilityInsights.SharedUx/Highlighting/ClearOverlayDriver.cs

```diff
@@ -63,17 +64,19 @@ public static void BringMainWindowOfPOIElementToFront()
                 NativeMethods.FocusWindow(handle);
                 ((IUIAutomationElement)SelectAction.GetDefaultInstance().POIElementContext.Element.PlatformObject).SetFocus();
             }
-            catch (ArgumentException)
-            {
-            }
-            catch(COMException)
-            {
-            }
-            catch (NullReferenceException)
-            {
-            } 
-            catch (InvalidOperationException)
+            catch (Exception e)
             {
+                if (e is ArgumentException ||
+                    e is COMException ||
+                    e is NullReferenceException ||
+                    e is InvalidOperationException)
+                {
+                    e.ReportException();
```

Comment:
Same question as before about only reporting the exceptions we aren't throwing

Reply:
Same as above. We only want to call ReportExceptions when we eat the Exception, and want to avoid multiple counts